src/components/NcRelatedResourcesPanel/NcResource.vue

```diff
@@ -77,9 +88,19 @@ export default {
 
 	data() {
 		return {
+			icon: subtitleIconMap[this.subtitle],
 			labelTranslated: t('Open link to "{resourceTitle}"', { resourceTitle: this.title }),
 		}
 	},
+
+	computed: {
+		isDarkTheme() {
+			return this.colorMainText === '#D8D8D8'
```

Comment:
this will certainly break at some point :see_no_evil:

Reply:
See comments in https://github.com/nextcloud/nextcloud-vue/pull/3236#pullrequestreview-1105320756 for solution